src/Microsoft.VisualStudio.AppDesigner/PropPages/PropPageUserControlBase.vb

```diff
@@ -812,7 +812,7 @@ Namespace Microsoft.VisualStudio.Editors.PropertyPages
                 If ProjectProperties IsNot Nothing Then
                     Return ProjectProperties.OutputType
                 Else
-                    Dim obj As Object = TryGetNonCommonPropertyValue(GetPropertyDescriptor("OutputType"))
+                    Dim obj As Object = TryGetNonCommonPropertyValue(GetPropertyDescriptor("OutputTypeEx"))
```

Comment:
As @davkean has mentioned before, I'm concerned about functional changes to the editor. Is this an actual bug in the implementation, or will this change behavior for non-core apps?